studio-docs/source/sharing-graphs.mdx

```diff
@@ -0,0 +1,402 @@
+---
+title: Sharing your graph
+---
+
+import {
+  ExpansionPanel,
+} from 'gatsby-theme-apollo-docs/src/components/expansion-panel';
+import { MultiCodeBlock } from 'gatsby-theme-apollo-docs';
+import {EmbeddableExplorer} from 'gatsby-theme-apollo-docs';
+
+Apollo Studio provides features that help you share your graph with both teammates and external consumers.
+
+## Customizing a graph's homepage
+
+When a user opens one of your graph's variants in Apollo Studio, its **homepage** is shown:
+
+Like a typical Git repository's homepage, this page displays a Markdown-based README that you can edit with any content you like.
+
+> **Note:** Only organization members with the [`Org Admin`, `Graph Admin`, or `Contributor` role](./org/members/) can edit a variant's README. If a variant is [protected](./org/graphs/#protected-variants-enterprise-only), `Contributor`s _cannot_ edit its README.
+
+**Each variant has its own README.** If you edit one variant's README, those edits are _not_ applied to your graph's other variants. Among other uses, this enables you to provide different information to external consumers in a [public variant](#making-a-graph-public).
+
+### README shortcodes
+
+The README supports a collection of variables that you can include in its text. These variables are rendered with the correct value for the current variant.
+
+
+
+## Public graphs
+
+You can make individual [variants](./org/graphs/#managing-variants) of your graph public. If you do, **anyone with the link** can view the following pages for that variant in Studio:
```

Comment:
@StephenBarlow anyone with the link seems scary. Maybe we can modify to say _anyone with the link to the public variant_? We may also want to add that the link will have a /public/ in the URL?